boards/raspberrypi/common/rpi_pico-pinctrl-common.dtsi

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021, Yonatan Schachter
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* The Pico and Pico 2 are pin compatible. */
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_P0>;
+		};
+		group2 {
+			pinmux = <UART0_RX_P1>;
+			input-enable;
+		};
+	};
+
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <I2C0_SDA_P4>, <I2C0_SCL_P5>;
+			input-enable;
+			input-schmitt-enable;
+		};
+	};
+
+	i2c1_default: i2c1_default {
+		group1 {
+			pinmux = <I2C1_SDA_P6>, <I2C1_SCL_P7>;
+			input-enable;
+			input-schmitt-enable;
+		};
+	};
+
+	spi0_default: spi0_default {
+		group1 {
+			pinmux = <SPI0_CSN_P17>, <SPI0_SCK_P18>, <SPI0_TX_P19>;
+		};
+		group2 {
+			pinmux = <SPI0_RX_P16>;
+			input-enable;
+		};
+	};
+
+	pwm_ch4b_default: pwm_ch4b_default {
+		group1 {
+			pinmux = <PWM_4B_P25>;
+		};
+	};
+
+	adc_default: adc_default {
+		group1 {
+			pinmux = <ADC_CH0_P26>, <ADC_CH1_P27>, <ADC_CH2_P28>, <ADC_CH3_P29>;
+			input-enable;
+		};
+	};
+
```

Comment:
```suggestion
```

Reply:
What's the purpose of the suggestion here?

Reply:
remove the empty line

Reply:
The **file** to end in a newline, this isn't the end of the file, there is another bracket below it. You tell me why you think lines 54-55 should be:
```
		};
	};
```
but lines 55-57 should be:
```
	};

};
```
?

Reply:
These files were updated in 2022 and no changes have been made since then, but our customs and norms have been updated in the meantime. Verbalized regulations are not everything. Customs and norms are also rules, and they are applied gradually through review. We should not pick up exceptional cases and rationalize them.

Reply:
Ah, sorry, I misread this as line 56, not 58.

In general, my approach when moving a file is "move it, don't move and then edit it." on the assumption that if was ok before, it should still be ok if all I've done is moved it. 

If the policy is "fix things if you touch them", then that's different, and does risk making PRs more and more work.

Reply:
In generally, we follows the "fix things if you touch them" rule. It came from simple reason, because we review the sources if changed. 
Zephyr has a comprehensive CI system, but that doesn't mean all problematic code caught, and if problematic code is discovered during review, it will need to be fixed.

Reply:
I think this is a fair ask, it's not like you just renamed the file and it's not like the request is to introduced any functional change that has to be tested again.